Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:18-alpine as builder
+
+WORKDIR /app
+
+COPY . .
+
+RUN npm install && \
+	npx next build && \
+	npx next export
```

Comment:
Can add another line to `package.json` called `build:ci` or something that just runs `next export`

Reply:
And then use that here